content/en/profiler/enabling/dotnet.md

```diff
@@ -134,9 +135,10 @@ To install the .NET Profiler per-application:
 </div>
 
 To install the .NET Profiler per-webapp:
-1. Install the Azure App Service [Datadog APM Extension][1] to your webapp.
+1. Install Azure App Service [Windows Datadog APM Extension][1] or follow [the Linux setup][2] for your webapp.
```

Comment:
```suggestion
1. Install the Azure App Service Datadog APM Extension [for Windows]1 or use the [Linux setup][2] for your webapp.
```

Reply:
Shouldn't it be `[for Windows][1]` instead of `[for Windows]1` ?
@buraizu